dartsim/src/SDFFeatures_TEST.cc

```diff
@@ -972,10 +972,10 @@ TEST_P(SDFFeatures_TEST, Shapes)
   auto dartWorld = world->GetDartsimWorld();
   ASSERT_NE(nullptr, dartWorld);
 
-  ASSERT_EQ(5u, dartWorld->getNumSkeletons());
+  ASSERT_EQ(6u, dartWorld->getNumSkeletons());
 
   int count{0};
-  for (auto name : {"sphere", "box", "cylinder", "capsule", "ellipsoid"})
+  for (auto name : {"sphere", "box", "cylinder", "capsule", "ellipsoid", "cone"})
```

Comment:
```suggestion
  for (auto name : {"sphere", "box", "cylinder", "capsule", "ellipsoid",
                    "cone"})
```
linter complains about line length from one of my patches

Reply:
@scpeters no worries, I also changed the number of overall segments in https://github.com/gazebosim/gz-physics/pull/639/commits/642692058a5bc51e2cc681caeb701197a9349ba3 to what we had discussed in https://github.com/gazebosim/gz-physics/pull/639#discussion_r1605419363